shared/AppInsightsCore/Tests/Perf/src/CorePerfCheck.Tests.ts

```diff
@@ -143,7 +176,7 @@ export class CorePerfCheckTests extends AITestClass {
             name: "PerfChecks: objForEachKey dynamic large (40 fields) object",
             timeout: 60000,
             test: () => {
-                let iterations = 100000;
+                let iterations = 90000;
```

Comment:
Reducing iterations as the perf tests sometimes timeout on the official build environments